go.mod

```diff
@@ -5,51 +5,42 @@ go 1.18
 require (
 	github.com/filecoin-project/go-data-transfer v1.15.2
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/ipfs/go-cid v0.2.0
-	github.com/ipfs/go-datastore v0.5.1
-	github.com/ipfs/go-graphsync v0.13.2
+	github.com/ipfs/go-cid v0.3.2
+	github.com/ipfs/go-datastore v0.6.0
+	github.com/ipfs/go-graphsync v0.13.1
```

Comment:
Is there a need to revert to a previous version of go-graphsync?

Reply:
this was the same as what boost / other rebasing against the newer libp2p were using and i figured i'd match that